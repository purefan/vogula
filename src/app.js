import React from 'react';
import {BrowserRouter as Router, Route } from 'react-router-dom';
import {Provider} from 'react-redux';
import store from './store';
// Styles
import './app.css';
// Components
import Header from './components/header';
import Home from './containers/home';
import AddConnection from './containers/add-connection';
import EditConnection from './containers/edit-connection';

class App extends React.Component {

	render() {
		return (
			<Provider store={store}>
				<Router>
				<div className="App">
					<Header />
					<Route exact path="/" render={() => <Home />} />
					<Route path="/add" render={() => <AddConnection />} />
					<Route path="/manage" render={() => <EditConnection />} />
				</div>
				</Router>
			</Provider>
		);
	}
}

export default App;
