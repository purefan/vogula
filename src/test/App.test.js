import React from 'react';
import renderer from 'react-test-renderer';

import App from '../app';

it('renders without crashing', () => {
  const app = renderer.create(<App />);
  expect(app).toMatchSnapshot();
});
