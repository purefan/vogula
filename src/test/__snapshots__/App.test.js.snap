exports[`test renders without crashing 1`] = `
<div
  className="App">
  <div
    className="Header">
    <span
      className="logo">
      vogula
    </span>
    <a
      className="button pull-right"
      href="/manage"
      onClick={[Function]}>
      Edit Connections
    </a>
    <a
      className="button pull-right"
      href="/add"
      onClick={[Function]}>
      Add Connection
    </a>
    <a
      className="button pull-right"
      href="/"
      onClick={[Function]}>
      Dashboard
    </a>
  </div>
  <div
    className="Home">
    <h3
      className="no-connection">
       No Connections 
    </h3>
  </div>
</div>
`;
